### ISSUE TITLE
Restore codeFlowLocation.message and stackFrame.message

### ISSUE THREAD

**Issue by lgolding** [oasis-tcs/sarif-spec#140]
When we did the location redesign (#130), we removed `annotatedCodeLocation.message`, because we reasoned “`codeFlowLocation` now has a `location` property, and the `location` object has a `message` property, so we don’t need `codeFlowLocation.message`.”

But that was wrong.  The same code flow can hit the same location multiple times. The same is true for `stackFrame`, if the stack has a recursive call.

**Comment by michaelcfanning** [comment]
@lgolding 
What am I missing here, won't each codeFlow in this case have its own location member/instance, which could hold a unique message?

**Comment by lgolding** [comment]
@michaelcfanning Yes. Recall the long thread we just had about this: I opened this issue because I was concerned about two things:

1. Is it ok to have a `location` that contains nothing but a `message`? We decided the answer was "yes", so this is no longer a concern.

2. Might multiple `codeFlowLocations` reference the same `location`, so `location.message` would not allow each `codeFlowLocation` to have a unique `message`? I realized I was silly: a `codeFlowLocation` does not _refer to_ a _location_; it _contains_ (embeds) a location. So this is also no longer a concern.

Bottom line: I have closed this issue.
